### DISCUSSION TITLE
Key points of next identity design

### DISCUSSION THREAD

**Discussion by hackfisher** [darwinia-network/darwinia#1406] [Ideas]
As the DApps, wallets and infrastructures in EVM+ ecosystem are more standardized in ENS design, we should make the next design of identity more friendly with  ENS.

1. We should encourage and leverage more application level offchain ENS/DNS-CCIP integration usage. The applications can be registrars themselves.
2. Deprecate and remove substrate identity pallet, try migrate existing identity data to other on-chain identity solutions which are friendly with ENS, probably implemented in smart contracts. (1.0 -> 2.0 migration can just remove the identity migration or emit events for script migration with the prerequisite that those un-migrated identities are reserved in new on-chain  solution)
3. Following 2, research of Darwinia/L2 support integration with ENS is still TBD, refer https://docs.ens.domains/dapp-developer-guide/ens-l2-offchain

Ref:
https://warpcast.com/hackfisher.eth/0xa799e4c2

**Comment by aurexav** [comment]
For Polkadot there is a people chain now.

**Comment by hackfisher** [comment]
https://github.com/ringecosystem/collaboration/pull/83
